run/h2c/README.md

```diff
@@ -0,0 +1,59 @@
+# Cloud Run HTTP/2 Sample Server
+
+This Go application serves requests using only the HTTP/2 cleartext (h2c)
+protocol (it does not support upgrading from HTTP/1.1).
+
+It is provided for testing end-to-end HTTP/2 capabilities on Cloud Run.
+
+## Deploying to Cloud Run
+
+Click this button:
+
+[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run)
+
+Alternatively, you can use the `gcloud` SDK:
+
+```sh
+gcloud beta run deploy h2c --source=. --use-http2
+```
+
+**Cleanup:** Remove the `h2c` Service you deployed from Cloud Run
+using the [Cloud Console](https://console.cloud.google.com/run).
+
+## Testing locally
+
+1. Check out this repository and navigate to this directory
+
+1. Start the server locally:
+
+    ```sh
+    go run .
+    ```
+
+## Verifying h2c
```

Comment:
In general we don't have README's unless the particular sample requires it, in preference to maintaining instructions only in the official docs. I'm not sure if this section would be added to the docs, so I'm in favor of keeping this README and keeping it here at least until the scope of HTTP2 content in the Cloud Run docs is settled.